### ISSUE TITLE
[Bug Report]: tailwindcss does not work on re-rendered lazy-loaded components in dev mode

### ISSUE THREAD

**Issue by MacbClarke** [farm-fe/farm#2024]
### Steps to reproduce

if a component is lazy-loaded and re-rendered after changing the tailwindcss classname, it won't work.

e.g. 
change `gap-1` to `gap-2`, `gap-2` has no effect
change back to `gap-1` works still.

now if i leave `gap-2` unchanged and restart the farm dev server, it works and `gap-1` does not.

### What is actually happening?

tested with vite with the same configs, it works well.

farm.config.ts
```typescript
import { defineConfig } from "@farmfe/core";
import farmJsPluginPostcss from "@farmfe/js-plugin-postcss";

export default defineConfig({
  plugins: ["@farmfe/plugin-react", farmJsPluginPostcss()],
  compilation: {
    sourcemap: false,
  },
});
```

postcss.config.js
```javascript
module.exports = {
  plugins: {
    tailwindcss: {},
    autoprefixer: {},
  },
};
```

tailwind.config.js
```javascript
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
  theme: {
    extend: {},
  },
  plugins: [],
};
```

### System Info

```Shell
archlinux in wsl2
```

**Comment by fu050409** [comment]
Thanks, it's a known bug and we'll fix it as soon as we can.

Now farm supports to use tailwindcss plugin directly, consider have a try with `@farmfe/js-plugin-tailwindcss`?

**Comment by MacbClarke** [comment]
@fu050409 Happy to try!
Is there any documentation on it?

**Comment by fu050409** [comment]
@MacbClarke You can see https://github.com/farm-fe/farm/blob/main/examples/tailwind-next/farm.config.ts as an example :-D

**Comment by MacbClarke** [comment]
@fu050409
Spent some time tinkering with `@farmfe/js-plugin-tailwindcss`, but it still feels a bit buggy, the styling doesn't look right, or sometines it works and sometimes it doesn't. Maybe i messed up somewhere in the config? Not sure.

For now, i'll just stick with Vite.
Hope you guys can fix this issue soon and i would really love to switch back to farm cuz it is indeed super fast!

**Comment by wre232114** [comment]
@MacbClarke  Could you provide a repo that can reproduce your issue? We'll fix this issue ASAP

**Comment by fu050409** [comment]
@wre232114 A reprocution repo is created at <https://github.com/fu050409/farm-postcss-hmr>, take a look?

**Comment by wre232114** [comment]
Fixed in `@farmfe/core@1.6.2`
